cli/commands/remote/table/index.md

```diff
@@ -15,6 +15,7 @@ kbc remote table [command]
 |---
 | Command | Description
 |-|-|-
+| [kbc remote table create](/cli/commands/remote/create/table/) | Create a new [table](https://help.keboola.com/storage/tables/). |
```

Comment:
1. You moved the file.
2. But the link is not updated.
3. Also `permalink` [file header](https://raw.githubusercontent.com/keboola/developers-docs/1d1b03ed8e7df9bddbce2416c318670ef2ece513/cli/commands/remote/table/create/index.md) is not updated.
4. Navigation/menu update is also missing: https://github.com/keboola/developers-docs/blob/main/_data/navigation.yml